prow/knative/config.yaml

```
@@ -200,6 +200,10 @@ branch-protection:
         - tide
       exclude:
       - "^dependabot/" # don't protect branches created by dependabot
+      include:
+      - "^main$"
+      - "^master$"
+      - "^release-*"
```

I think the above regex is missing a `.` 

```suggestion
      - "^release-.*"
```